lean-client-js-browser/README.md

```diff
@@ -1,3 +1,48 @@
+
+The main class is `WebWorkerTransport`. The constructor takes [a `LeanJsOptions` object](src/inprocesstypes.ts) with the following fields:
+  - `libraryZip`: URL to `library.zip`. Must end in `.zip`. This field is used to generate the default values for some of the optional fields below.
+  - `libraryMeta` (optional): URL to `library.info.json`, a JSON file whose keys are the Lean packages contained in `library.zip` and whose values are URL prefixes that point to the Lean source files. (default: the value of `libraryZip` with the final `.zip` replaced with `.info.json`)
+  - `libraryOleanMap` (optional): URL to `library.olean_map.json`, a JSON file whose keys are the filenames of the oleans in `library.zip` (with .olean removed) and whose values are the name of the Lean package that they originate from (default: the value of `libraryZip` with the final `.zip` replaced with `.olean_map.json`)
+  - `libraryKey` (optional): name for the key used in the indexedDB cache (default: the filename of `libraryZip` with `.zip` removed)
+  - At least one of `javascript` or (`webassemblyWasm` + `webassemblyJs`) must be provided:
+    - `javascript`: URL to `lean_js_js.js`
+    - Both of the following fields should be provided to use the WebAssembly version of the Lean server:
+      - `webassemblyWasm`: URL to `lean_js_wasm.wasm`
+      - `webassemblyJs`: URL to `lean_js_wasm.js`
+  - `memoryMB` (optional): size of memory in MB provided (default: 256)
+  - `dbName` (optional): name of the IndexedDB database used to cache (default: `leanlibrary`)
+
+The `WebWorkerTransport` object that is returned should be passed to the `Server` constructor. See the `lean-client-js` library for more information about the `Server` class.
+
+See [`demo.ts`](./demo.ts) for an example on how to use this package in a TypeScript project. After building, see `dist/lib_test.html` for how to use this package in a webpage.
+
+See [`leanprover-community/lean-web-editor`](https://github.com/leanprover-community/lean-web-editor) and [`mpedramfar/Lean-game-maker`](https://github.com/mpedramfar/Lean-game-maker) for example TypeScript projects that use this library.
+
+See [these Observable notebooks](https://observablehq.com/collection/@bryangingechen/lean) for example webpages that use the UMD module `leanBrowser.js`.
+
+## Building
+
+To build the demo file, follow the directions in the README file in the parent package `lean-client-js`. The demo file will be written to `dist/demo.html`.
+
+Running `../node_modules/.bin/webpack` from this directory will build and output the test file `dist/lib_test.html` as well as a UMD module `dist/leanBrowser.js` for use in webpages.
+
+Once the files are built, you can check them out by starting a local web server (from the `dist/` directory) and navigating to `/index.html` or `/lib_test.html`.
+
+## Changes in vXXX
+
+This version of `lean-client-js-browser` has two main differences from versions before vXXX:
```

Comment:
These `XXX`s should be replaced with the version number of the release containing these changes (if they get merged).

Reply:
I wouldn't put too much effort in release notes with version numbers (but feel free to put in the version number you think it deserves).  I don't think anybody is using `lean-client-js-browser` as a library.  And this PR has already been in production use without anybody being confused for what, a year now?

Reply:
OK, makes sense. I do know that @mpedramfar [has been using my fork of `lean-client-js-browser`](https://github.com/mpedramfar/Lean-game-maker/blob/master/src/interactive_interface/package.json#L12) in the Lean-game-maker.

I guess if we want to abide by semver, this should be a major version bump, so 2.0.0?

Reply:
2.0.0 works for me.